packages/openapi-v3-types/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017,2018. All Rights Reserved.
```

Comment:
I think as a new package created in `2018` the header should be just `2018` and not `2017,2018` but I'm not entirely sure about this

Reply:
Miroslav suggests to fix them by a command, running it, let's see.

Reply:
should be `2018` only 👍 an awesome command...updated 290+ files, probably create another PR to update them.